tests/mypy/configs/pyproject-plugin-bad-param.toml

```diff
@@ -29,6 +29,13 @@ check_untyped_defs = true
 no_implicit_reexport = true
 disallow_untyped_defs = true
 
+python_version = '3.10'
+
+# 3.9 drop the following line:
+force_uppercase_builtins = true
+# 3.10 drop the following line:
+force_union_syntax = true
+
```

Comment:
There's a lot of reuse here - is there a more efficient way to include these settings in the config files? Any concepts of inheritance / base configs we could use? Just spitballing here, haven't looked into this at all...

Reply:
mypy doesn't, but the `pytest-mypy-plugins` library does